agents/locobot/teleop.py

```diff
@@ -231,78 +270,83 @@ def test_command(sid, commands, yaw_velocity):
-        mover.explore()
+        if backend == 'habitat':
+            mover.explore()
         
-        # get the SLAM goals
-        goal_loc, stg = None, None # mover.bot.get_slam_goal()    
-
-        # plot the final goal
-        if goal_loc is not None:
-            goal_x, goal_y, goal_z = goal_loc
-            cone = o3d.geometry.TriangleMesh.create_cylinder(radius=.2,
-                                                             height=3.,)
-            cone.translate([goal_x, goal_y, 0.4], relative=False)
-            cone.compute_vertex_normals()
-            cone.paint_uniform_color([0.0, 1.0, 1.0])
-            o3dviz.put('goal_cone', cmd, cone)
-
-        # plot the short term goal in yellow and the path in green
-        if stg is not None:
-            stg_x, stg_y = stg
-            cone = o3d.geometry.TriangleMesh.create_cylinder(radius=.2,
-                                                             height=3.,)
-            cone.translate([stg_x, stg_y, 1.4], relative=False)
-            cone.compute_vertex_normals()
-            cone.paint_uniform_color([1.0, 1.0, 0.0])
-            o3dviz.put('stg', cmd, cone)
-
-            if prev_stg is None:
-                prev_stg = [y, -x]
-            cur_stg = [stg_x, stg_y]
-
-            arrow_length = distance.euclidean(cur_stg, prev_stg)
-            if arrow_length > 0.0001:                
-                path = o3d.geometry.TriangleMesh.create_arrow(cylinder_radius=.03,
-                                                              cone_radius=.04,
-                                                              cylinder_height = arrow_length / 2,
-                                                              cone_height = arrow_length / 2,)
-                path.compute_vertex_normals()
-                path.paint_uniform_color([0.0, 1.0, 0.0])
-        
-                path.translate([prev_stg[0], prev_stg[1], 0.2], relative=False)
-                path.rotate(o3d.geometry.get_rotation_matrix_from_axis_angle([0, math.pi/2, 0]))
-                path.rotate(o3d.geometry.get_rotation_matrix_from_axis_angle([0, 0, yaw]))        
-                o3dviz.put('short_term_goal_path_{}'.format(path_count), 'add', path)
-                path_count = path_count + 1
-            prev_stg = cur_stg
-
-        # # get the obstacle map and plot it
-        # obstacles = mover.bot.get_map()
-        # obstacles = np.asarray(obstacles)
-        # obstacles = np.concatenate((-obstacles[:, [1]], -obstacles[:, [0]], np.zeros((obstacles.shape[0], 1))), axis=1)
-        # obspcd = o3d.geometry.PointCloud()
-        # obspcd.points = o3d.utility.Vector3dVector(obstacles)
-        # obspcd.paint_uniform_color([1.0, 0., 0.])
-        # obsvox = o3d.geometry.VoxelGrid.create_from_point_cloud(obspcd, 0.03)
-        # o3dviz.put('obstacles', cmd, obsvox)                
-            
+            # get the SLAM goals
+            goal_loc, stg = None, None # mover.bot.get_slam_goal()    
```

Comment:
hook up the explore to a command as well (like 'explore')? this is a no-op right now as I understand and needs to be uncommented to work, right?

Reply:
will make this work again and uncomment in a subsequent PR